src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,11 +1,74 @@
 package core.basesyntax;
 
-/**
- * <p>Дано класс Plane. Написать класс PlaneBuilder, который реализует паттерн Builder.
- * Добавьте не менее пяти полей в класс Plain. Внести все необходимые изминения в класс Plane
- * которые могут потребоваться для реализации паттерна Builder.</p>
- */
-
 public class Plane {
+    private final String name;
+    private final String type;
+    private final int maxSpeed;
+    private final int passengerCapacity;
+    private final int rangeOfFlight;
```

Comment:
why are you using `final` here?

Reply:
Honestly, because idea suggested to do so. And I've read about it and decided that it is a good practice to make variables final if we have no setters for them. But first of all because of idea. 
I removed final, but now I don't understand, should I use it here or not. Can you explain please?

Reply:
@OleksiiTromsa `final` is used if we don't want the variable to point at any other value in the future
in this case we don't have setters, and our fields are private - so, they cannot be changed - and there is no need to additionally make them final

you will learn more about `final` keyword soon when studying immutable class